app/views/devise/_links.erb

```diff
@@ -0,0 +1,3 @@
+<% ActiveSupport::Deprecation.warn "Rendering partials devise/_links.erb is deprecated" \
+  "please use devise/shared/_links.erb instead."%>
+<%= render "shared/links" %>
```

Comment:
Is "devise/_links.erb" rendered automatically by devise? If not, why do we need this file? Can't we just render "shared/links"